apps/vh/test-api/prod.yaml

```diff
@@ -9,7 +9,6 @@ spec:
     java:
       ingressHost: vh-test-api.platform.hmcts.net
       environment:
```

Comment:
this is missing a lot of environment variable overrides.

Reply:
will complete in new PR